traits/Cargo.toml

```diff
@@ -13,6 +13,7 @@ path = "./traits.rs"
 
 [dependencies]
 failure = "0.1.1"
+gimli = "0.16.0"
```

Comment:
This dependency should be optional, and only enabled when the parser has the `dwarf` feature enabled.